portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,54 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+    private ArrayList<String> comments = new ArrayList<String>();
+    
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    
+    synchronized(comments){
+       comments.add("Hello World!");
+       comments.add("Hola Mundo!");
+       comments.add("Hallo Welt!");
+       response.setContentType("application/json;");
+       String json = convertToJson(comments);
```

Comment:
Can you use the gson.toJson instead since you've already added the dependency? Delete the hard-coded convertToJson method.

Reply:
Ohh wait I see you're using the same ArrayList every time.